static/locales/fr.client.json

```diff
@@ -1946,5 +1946,23 @@
         "Have you **reviewed the code** at this URL?": "Le cas échéant avez-vous pu **auditer le code derrière ce lien** ?",
         "If in doubt, do not install this widget, or ask an administrator of your organization to review it for safety.": "En cas de doute, vous pouvez transférer le lien aux équipes techniques responsable de votre instance.",
         "I confirm that I understand these warnings and accept the risks": "j’ai bien lu ces recommandations et je confirme vouloir ajouter cette vue"
+    },
+    "NewRecordButton": {
+        "New card": "Nouvelle fiche",
+        "New record": "Nouvelle ligne"
+    },
+    "Experiments": {
+        "{{- experiment}} disabled.": "{{- experiment}} : désactivé.",
```

Comment:
Same as for the other PR, I fear that Weblate will be lost if we start translating strings from the repo.

Reply:
I'll just remove those and go through weblate later :+1: